javascript/GridFieldExtensions.js

```diff
@@ -1,3 +1,8 @@
+function GFEremoveChanged() {
```

Comment:
For entwine usage, it'll be something like 

```
$('css-selector-for-the-SaveRow-button').entwine({
    click: function (e) {

    }
});
```

Or more simply, you can just do it using a jquery "on" event handler

```
$(document).on('click', 'css-selector-for-the-SaveRow-button', function (e) {

});
```